public/emoji/.gitignore

```diff
@@ -0,0 +1 @@
+*_border.svg
```

Comment:
My idea was to have the generated files in the repo, like we may do with the lock files, or the `emoji_map.json`.